wordpress/wp-content/themes/cds-default/footer.php

```diff
@@ -14,7 +14,7 @@
         <nav class="container wb-navcurr">
             <h2 class="wb-inv"><?php echo $footerText['aboutGovernmentTitle']; ?></h2>
             <ul class="list-unstyled colcount-sm-2 colcount-md-3">
-                <?php echo print_menu_links($footerText['aboutGovernment']); ?>
+                <?php echo print_footer_links($footerText['aboutGovernment']); ?>
```

Comment:
I know this is an existing function name now but seems a little odd to say "echo print" maybe move the echo inside the function as we'll know it's meant to print.

Reply:
Maybe `get_footer_links` is more reasonable? 
Or some of the PHP-y methods are like `wp_header()`, so this could just be `footer_links()`?